### ISSUE TITLE
"URL-Fehler" in Kombination mit iPad - The request timed out

### ISSUE THREAD

**Issue by V-Pixel1** [ioBroker/ioBroker.vis#523]
**Describe the bug**  
Every time the iPad is woken up from the "standby" state and thus the VIS should be displayed automatically, the Main-View appears and an error message with the text "Error: URL error: Please check the URL: http://192.168.178.51:8082/vis/index.html?main#Main 
The request timed out."

After confirming this window, the VIS continues to run as usual. This error is very often but not every time." Same issue have some other Users with iPads.

**Screenshots & Logfiles**  
![visfehler](https://user-images.githubusercontent.com/77643163/188280593-09df6459-52b0-4e48-b226-25a6515c7345.jpeg)

Log: "web.0: ==> Connected system.user.admin from ::ffff:192.168.178.69"

**Versions:**
 - Adapter version: v1.4.15
 - JS-Controller version: 4.0.23
 - Node version: v16.17.0
 - Operating system: Raspbian

**Additional context**

**Comment by Knarfen90** [comment]
I’ve got the same issue. Did you find a solution in meanwhile?

**Comment by V-Pixel1** [comment]
> I’ve got the same issue. Did you find a solution in meanwhile?

No, I‘ve changed iPad to a Samsung Galaxy Tab A, works very well now.
